xml/ay_registration_extensions.xml

```diff
@@ -421,6 +421,10 @@ x509 -noout -fingerprint -sha256</screen>
    be specified in the profile, including the registration key. Otherwise the
    registration would fail.
  </para>
+  <para>
+   Architecture and version of an extension is not mandatory. The registration
```

Comment:
Looks fine. Perhaps it might be a little clearer if we said:

"Architecture and version of extensions are not mandatory." 

Just to be explicit that we're not talking about any particular extensions.